### PR TITLE
Update to pants 1.4.0rc0

### PR DESCRIPTION
**Pull request by stuhood**

### CODE REVIEW

File: pants.ini [hunk 1]
```diff
@@ -21,14 +21,15 @@ pythonpath: [
   ]
 backend_packages: +[
     'twitter.common.pants.python.commons',
-    'pants.contrib.python.checks',
```
Comment (kwlzn) [inline pants.ini]: intentional?

Comment (stuhood) [reply pants.ini]: Yes: Putting something in the plugin list enables it, so having this here was enabling it twice, which now causes an error.

Comment (kwlzn) [reply pants.ini]: ah, cool

**Comment by stuhood** [comment]
As evidenced by one shard failing with a timeout at the end of pytest running, and the other not, it looks like this has nothing to do with the daemon. Regardless, will skip enabling it in this bump because I've already made enough edits.

**Pull request merged**